main.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"bufio"
 	"fmt"
+	"github.com/bitrise-io/go-utils/env"
```

Comment:
nit: Import order. [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports?utm_source=godoc) can help enforce a consistent import order, it's also available in GoLand as a File Watcher option

Reply:
`goimports` or GoLands built in import sorting is the default in our team? 

GoLand simply sorts it alphabetically while `goimports` creates groups and sorts the imports in the groups alphabetically.

Reply:
I *think* we have an agreement to use `goimports`, not sure how official is that :)